packages/shared/styles/components/molecules/SfComponentSelect.scss

```diff
@@ -53,13 +53,14 @@
     color: var(--component-select-label-color, var(--c-secondary-variant));
     transform: var(
       --component-select-label-transform,
-      translate3d(0, calc(-1 * var(--component-select-label-top, 50%)), 0)
```

Comment:
question: same as above, isn't this a breaking change?

Reply:
Like above.